data/fixtures/scopes/python/collectionItem.unenclosed.iteration3.scope

```diff
@@ -2,10 +2,25 @@ def foo():
     global bar, baz
 ---
 
-[Range] = 1:11-1:19
+[#1 Range] =
+[#1 Domain] = 0:8-0:8
+          ><
+0| def foo():
```

Comment:
NB: this range is technically empty. We don't consider it worthwhile to omit empty ranges from this test file.

Andreas says: conceivably it could be useful to be able to boot strew up an empty list by saying "pour item" and targeting this range, so empty ranges are not necessarily useless.